lectures/views.py

```diff
@@ -1,3 +1,95 @@
-from django.shortcuts import render
+import boto3
 
-# Create your views here.
+from uuid          import uuid4
+from django.http   import JsonResponse
+from django.views  import View
+
+from users.models    import User
+from lectures.models import Lecture, LectureImage
+from core.decorator  import signin_decorator
+from my_settings     import AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY
+
+class FileView(View):
```

Comment:
```suggestion
class LecturesView(View):
```
수업과 관련된 view이기 때문에 다음과 같이 이름을 변경하는 것이 좋을 것 같습니다.